FreeRTOS/FreeRTOS-Kernel
# Update MPU wrapper for pcTaskGetName API

Description
-----------
This PR makes MPU_pcTaskGetName privileged only. This is done because it returns a pointer from TCB which an unprivileged cannot de-reference. 

Test Steps
-----------
Run soak tests on ARMv7M and ARMv8M ports.

Checklist:
----------



Related Issue
-----------
 https://forums.freertos.org/t/questions-about-mpu-implement-on-armv8-m/17915


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
